scripts/create_amy_epw_files.py

```diff
@@ -537,12 +554,12 @@ def write_epw(save_path):
         # Write new EPW file.
         write_epw(amy_epw_file_out_path)
 
-    except:
+    except Exception as e:
         # Do the following if unable to complete the above process and convert to CSV.
         # TODO add in something to return the error
         no_epw.loc[no_counter, 'file'] = station_year
         no_counter += 1
-        print('Problem processing: ' + station_year)
+        print('Problem processing: ' + station_year + ': ' + str(e))
```

Comment:
I added this bit to make it easier to see why a record failed. I think I can fix the errors I'm seeing, but I'll save that for a later update.